src/definitions.js

```diff
@@ -571,6 +571,16 @@ const definitions = [
     version: '1',
     type: 'String',
   },
+  {
+    identifier: 'cvc:Type:timestamp',
+    version: '1',
+    type: 'String',
+  },
+  {
+    identifier: 'cvc:Type:simpleName',
```

Comment:
Why we have the simpleName? Just to write down a full string without first, middle and last?